rambo/app.py

```diff
@@ -323,7 +324,18 @@ def install_plugins(force=None, plugins=('all',)):
             if not force:
                 click.confirm('The plugin "%s" is not in our list of plugins. Attempt '
                           'to install anyway?' % plugin, abort=True)
-            _invoke_vagrant('plugin install %s' % plugin)
+            vagrant_general_command('plugin install %s' % plugin)
+
+def scp(ctx=None, locations=None):
+    assert(len(locations)==2)
+
+
+    if ':' in locations[0]: # [0] is remote, fix [1] which is local
```

Comment:
I want these variable names (location) to be assgined useful names like local and remote. I don't want to have to keep think about this stuff in terms of list indexes.

Reply:
Or  rather than remote / local, copy from, copy to. Which ever makes more since

Reply:
:+1: